src/installer/hash.ts

```diff
@@ -2,14 +2,17 @@ import * as ghCore from "@actions/core";
 import * as crypto from "crypto";
 import * as fs from "fs";
 
-import { HttpClient } from "../util/utils";
-import { ClientFile } from "../util/types";
-import { getDirContents } from "../client-finder/directory-finder";
-import { isOCV3 } from "../client-finder/oc-3-finder";
+import { HttpClient, getAssetDownloadPath } from "../util/utils";
+import { ClientFile, InstallableClient } from "../util/types";
+import { getDirContents } from "../mirror-client-finder/directory-finder";
+import { isOCV3 } from "../mirror-client-finder/oc-3-finder";
+import { getReleaseAssets } from "../github-client-finder/repository-finder";
 
-const SHA_FILENAMES = [ "sha256sum.txt", "SHA256_SUM" ];
+const SHA_FILENAMES = [ "sha256sum.txt", "SHA256_SUM", "checksums.txt" ];
 type HashAlgorithm = "md5" | "sha256";
 
+// Few clients that are installed from github lacks hash file in their release assets
+const clientsWithNoShaFile: InstallableClient[] = [ "kam", "kamel", "opm", "s2i" ];
```

Comment:
should this info be stored in the client overrides object?

Reply:
yeah that would be better way :+1: